docs/privacy.md

```diff
@@ -0,0 +1,155 @@
+# badrap.io Privacy Policy
+
+Badrap Ltd develops and maintains this service. We process your
+personal data according to the General Data Protection Regulation (GDPR)
+of the European Union, as well as according to all other relevant
+legislation according to Finnish and EU law. In this privacy
+policy we explain our basis for processing your personal
+information, and inform you of your rights regarding your personal
+data.
+
+## Data controller contact details
+
+Badrap Oy\
+Teknologiantie 18 B\
+90590 Oulu\
+Finland\
+email: [asiakaspalvelu@badrap.io](asiakaspalvelu@badrap.io)\
```

Comment:
Not sure if we have asiakaspalvelu@badrap.io alias.. @evilon ?

Reply:
Good point. For the record: the email address is also used in the "old" testaaverkkosi.fi privacy policy.

<img width="676" alt="screen shot 2018-08-03 at 11 17 32" src="https://user-images.githubusercontent.com/19776768/43632483-f876bc12-970e-11e8-9b17-4cb30b1959ac.png">

Reply:
Switched to support@badrap.io.